SwiftDocCRender.docc/SwiftDocCRender.md

```diff
@@ -1,8 +1,4 @@
-# Swift-DocC-Render
-
-@Metadata {
```

Comment:
This is no longer needed?

Reply:
Not strictly necessary anymore, although we could add it back if we wanted to change the main display name.